### PR TITLE
Implement ps functionality for DBus API

### PR DESCRIPTION
**Pull request by AmartC**
There is one thing I am awaiting feedback on and afterwards I will submit a new PR. In this case, is a separate tty function needed? The useTTY flag I feel would come in handy here however, I can change it to use a tty function if necessary.

### CODE REVIEW

**Comment by baude** [comment]
please use a commit message with a title (like you do) and then expand on what the PR does subsequently.

File: atomic_client.py [hunk 1]
```diff
@@ -77,6 +77,10 @@ def images(self):
     def vulnerable(self):
         return self.dbus_object.VulnerableInfo(dbus_interface="org.atomic", timeout = 2147400)
 
+    @polkit.enable_proxy
+    def ps(self):
+        return self.dbus_object.Ps(dbus_interface="org.atomic", timeout = 2147400)
```
Comment (baude) [inline atomic_client.py]: this may have been discussed before, but why are we setting a timeout limit?

File: atomic_dbus.py [hunk 2]
```diff
@@ -264,6 +266,17 @@ def VulnerableInfo(self):
         self.atomic.set_args(args)
         return self.atomic.get_all_vulnerable_info()
 
+    # The Ps method will send back information that says
+    # whether or not an installed container image is vulnerable
```
Comment (baude) [inline atomic_dbus.py]: i thought atomic ps == docker ps ?

File: atomic_client.py (hunk 1 above)
Comment (AmartC) [inline atomic_client.py]: If the timeout limit is not set, then there is the possibility that there can be a timeout before the data is sent back and output.

Comment (baude) [inline atomic_client.py]: Yes of course, i understand that ... but that timeout seems arbitrary and therefore could also be problematic.  would martin's approach here make sense?

Comment (AmartC) [inline atomic_client.py]: possibly this is the timeout though set for the test python client however. in cockpit, the client doesn't timeout. stefw had recommended this timeout length for me.

**Pull request closed without merging**